src/sagemaker_inference/logging.py

```diff
@@ -10,10 +10,17 @@
 # distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+"""This module contains logging functionality."""
 from __future__ import absolute_import
 
 import logging
 
 
 def get_logger():
+    """Return a logger with the name "sagemaker-inference",
+    creating it if necessary.
+
+    Returns:
+        (logging.Logger): Instance of the Logger class.
```

Comment:
you don't need the parentheses around the type for "Returns" (and "Raises")

same below